src/components/Nav/Nav.scss

```
@@ -0,0 +1,19 @@
+.Nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+  background-color: rgb(0, 0, 0);
+  color: rgb(255, 255, 255);
+  ul {
+    display: flex;
+    li {
+      padding: 30px 10px;
+    }
+  }
+  .navIcon {
+    color: white;
+    i {
+      padding-left: 30px;
+    }
+  }
+}
```

- 항상 새로운 셀렉터를 시작할때는 한 칸 띄워주세요!
```suggestion
.Nav {
  display: flex;
  align-items: center;
  justify-content: space-around;
  background-color: rgb(0, 0, 0);
  color: rgb(255, 255, 255);
  
  ul {
    display: flex;
    
    li {
      padding: 30px 10px;
    }
  }
  
  .navIcon {
    color: white;
    
    i {
      padding-left: 30px;
    }
  }
}
```

---

CSS에 가독성 고려해야 하는 점 꼭 기억하겠습니다!!😆